### ISSUE TITLE
parking_lot_core 0.6.0 does not compile on ARM

### ISSUE THREAD

**Issue by fulmicoton** [Amanieu/parking_lot#163]
[Tantivy CI does not pass on ARM ](https://travis-ci.org/tantivy-search/tantivy/jobs/559684485) due to parking_lot_core.

The error looks like this.

```rust
error[E0425]: cannot find function `__errno_location` in module `libc`
   --> /cargo/registry/src/github.com-1ecc6299db9ec823/parking_lot_core-0.6.0/src/thread_parker/linux.rs:112:28
    |
112 |                     *libc::__errno_location() == libc::EINTR
    |                            ^^^^^^^^^^^^^^^^ not found in `libc`

```

**Comment by CryZe** [comment]
It fails on Android, not ARM.

**Comment by fulmicoton** [comment]
@Cryze Oops yes my bad!

**Comment by Amanieu** [comment]
It seems that the function is `__errno` instead of `__errno_location` on android.

**Comment by Amanieu** [comment]
Fixed in parking_lot_core 0.6.1

**Comment by fulmicoton** [comment]
Awesome!
